include/net/fota_download.h

```diff
@@ -112,6 +118,36 @@ typedef void (*fota_download_callback_t)(const struct fota_download_evt *evt);
  */
 int fota_download_init(fota_download_callback_t client_callback);
 
+/**@brief Start downloading the given file from the given host. Validate that the
```

Comment:
Try to keep the brief short with just one sentence. You can bring the second sentence to the description.